backend/core/src/listings/dto/listing.dto.ts

```diff
@@ -371,8 +375,8 @@ export class ListingCreateDto extends OmitType(ListingDto, [
   @Expose()
   @IsDefined({ groups: [ValidationsGroupsEnum.default] })
   @ValidateNested({ groups: [ValidationsGroupsEnum.default], each: true })
-  @Type(() => IdDto)
-  applicationMethods: IdDto[]
```

Comment:
Why create application-methods through POST `/listings` and cascading when we have this a separate endpoint `/applicationMethods`? Are application methods always listing specific?